### PR TITLE
adding empty project checker

### PR DESCRIPTION
**Pull request by tvay11**
Screenshot 2023-04-11 at 11 33 42 AM

I am adding a checker to get a 404 error on an empty manifest.

### CODE REVIEW

File: src/java/edu/slu/tpen/servlet/ProjectServlet.java [hunk 1]
```diff
@@ -85,15 +85,17 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
...
+                    if (redirect.contains(check)) {
+                        projID = parseInt(redirect.replace("/" + check, ""));
+                        String redirectURL = req.getContextPath() + "/transcription.html?projectID=" + projID;
+                        resp.sendRedirect(redirectURL);
+                }   else {
                     //System.out.println("Project 2");
                     projID = parseInt(req.getPathInfo().substring(1).replace("/", "").replace("manifest.json",""));
                     Project proj = new Project(projID);
-                    //System.out.println("Project 3");
+                    System.out.println("Project 3");
```
Comment (thehabes) [inline src/java/edu/slu/tpen/servlet/ProjectServlet.java:98]: Clean these commented and active System.out's, we would rather not deploy superfluous logs

File: src/java/textdisplay/Project.java [hunk 2]
```diff
@@ -105,6 +105,29 @@ public void addTool(String name, String url) throws SQLException {
...
+            int ans = rs.getInt(11);
+            if(ans>0){
+                return true;
+            }
+         }
+      } finally {
+            closeDBConnection(j);
+            closePreparedStatement(ps);
+      }
+      System.out.println("not working");
```
Comment (thehabes) [inline src/java/textdisplay/Project.java:128]: remove pls

**Review by thehabes — CHANGES_REQUESTED** [review]
Hi there,
The code is successful and looks good (but you have some System.outs to clean up).  It would be mergeable.  However,  we don't want to bring that .yaml file in.  You should be working on that in another branch.

**Review by thehabes — APPROVED** [review]
Thanks for the clean up, looks good to go.

**Review by markskroba — APPROVED** [review]
Worked for me

**Pull request merged**